<?xml version="1.0"?>

<launch>
  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    <rosparam file="$(find final_assignment)/param/costmap_common_params.yaml" command="load" ns="global_costmap"/>
    <rosparam file="$(find final_assignment)/param/costmap_common_params.yaml" command="load" ns="local_costmap"/>
    <rosparam file="$(find final_assignment)/param/local_costmap_params.yaml" command="load"/>
    <rosparam file="$(find final_assignment)/param/global_costmap_params.yaml" command="load"/>
    <rosparam file="$(find final_assignment)/param/base_local_planner_params.yaml" command="load"/>
    <rosparam file="$(find final_assignment)/param/move_base_params.yaml" command="load"/>

    <node pkg="final_assignment" type="wall_follow_service_m.py" name="wall_follower" />
   <node pkg="final_assignment" type="call_target" name="user_interface" output="screen" />
  <node pkg="final_assignment" type="target_reply" name="randon_position_generator" output="screen" />

  </node>
</launch>
